src/main/kotlin/com/hedvig/lookupservice/visma/web/LookupController.kt

```diff
@@ -0,0 +1,29 @@
+package com.hedvig.lookupservice.visma.web
+
+import com.hedvig.lookupservice.util.toReference
+import com.hedvig.lookupservice.visma.service.VismaService
+import com.hedvig.lookupservice.visma.web.dto.RealEstateDto
+import com.hedvig.lookupservice.visma.web.dto.RealEstateResponse
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("_/v1/lookup")
```

Comment:
I may be wrong but I think /_/ is an implicit v1
```suggestion
@RequestMapping("_/lookup")
```